overlord/devicestate/handlers_install.go

```diff
@@ -111,6 +113,17 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("cannot make run system bootable: %v", err)
 	}
 
+	// disable cloud-init by default (as it's not confined)
+	// TODO:UC20: 1. allow drop-in cloud.cfg.d/* in mode dangerous
+	//            2. allow gadget cloud.cfg.d/* (with whitelisted keys?)
```

Comment:
we will need allow drop in even for non-dangerous for the maas case, and that's where whitelisting maybe is useful

Reply:
Good point, I added that now too